app/models/service_provider.rb

```diff
@@ -47,6 +47,10 @@ def live?
     active? && approved?
   end
 
+  def piv_cac_available?
+    PivCacService.piv_cac_available_for_agency?(metadata[:agency])
```

Comment:
Why not read the `agency` attribute directly from the DB?

Reply:
Good question. I'll change that.